corehq/apps/users/static/users/js/web_users.js

```diff
@@ -136,15 +132,19 @@ hqDefine("users/js/web_users",[
         }
 
         $('.delete-request').on('click', function (e) {
-            handleDeletion($(this),
+            handleDeletion($(this), {
+                    id: $(this).data('id'),
+                },
```

Comment:
To make stickler happy
```suggestion
            handleDeletion($(this),
                {id: $(this).data('id')},
```